dubbo-rpc/dubbo-rpc-triple/src/main/java/org/apache/dubbo/rpc/protocol/tri/command/FrameQueueCommand.java

```diff
@@ -18,15 +18,25 @@
 package org.apache.dubbo.rpc.protocol.tri.command;
 
 import io.netty.channel.Channel;
-import io.netty.channel.ChannelHandlerContext;
 import io.netty.channel.ChannelPromise;
+import io.netty.handler.codec.http2.Http2StreamFrame;
 
-public abstract class QueuedCommand {
+public class FrameQueueCommand {
```

Comment:
I recommend that the different child cmd not be deleted, the design is used to mask the h2 data structure